test/runtests.jl

```diff
@@ -3,180 +3,186 @@ include("dependencies_for_runtests.jl")
 CUDA.allowscalar() do
 
 @testset "Oceananigans" begin
+
     if test_file != :none
+
+        # just testing a single file
         @testset "Single file test" begin
             include(String(test_file))
         end
```

Comment:
Isn't it simpler to say that one should set `TEST_GROUP=nothing` to run just one file? Rather than changing `runtests.jl`? Just wondering if it's worth making this file more complicated so that people don't have to provide a non-default `TEST_GROUP` which seems relatively easy...

Reply:
have a look at the latest?